components/tabs/stories/tabs-horizontal.stories.js

```diff
@@ -30,11 +30,57 @@ export default {
   }
 };
 
-export const Default = Template.bind({});
-Default.args = horizontalProps;
+const horiztonalTabsGroup = ({
+	...args
+  }) => {
+    return html `
+      ${Typography({
+        semantics: "heading",
+        size: "s",
+        content: ["Horizontal Default"],
```

Comment:
```suggestion
        content: ["Horizontal default"],
```

Would you update headings to use sentence case? Thank you!